main.go

```diff
@@ -576,6 +582,38 @@ func createDatabase(project, instance, dbName string, conv *internal.Conv, out *
 	return fmt.Sprintf("projects/%s/instances/%s/databases/%s", project, instance, dbName), nil
 }
 
+func UpdateDDLForeignKeys(project, instance, dbName string, conv *internal.Conv, out *os.File) error {
```

Comment:
I am wondering why is this function exported?

Reply:
this function will be moved to conversion package once merged with web api branch, but for now it makes to be consistent with current code so i updated it to lower case function.